packages/node/src/lib/feeds/feeds.ts

```diff
@@ -0,0 +1,16 @@
+import { WithHttp } from '../novu.interface';
```

Comment:
```suggestion
import axios from 'axios';
import { WithHttp } from '../novu.interface';
```
Even if not use, I believe it's required in order to work with `WithHttp`